### ISSUE TITLE
Throwing exception on expired TTL for getSync is not very ergonomic

### ISSUE THREAD

**Issue by ejc3** [philcockfield/file-system-cache#35]
I'm curious why when getSync is called, the program throws when the value is expired:

```
export const toGetValue = (data: any) => {
  if (isExpired(data)) throw new Error(`Cache item has expired.`);
  if (data.type === 'Date') return new Date(data.value);
  return data.value;
};

  public getSync(key: string, defaultValue?: any) {
    const path = this.path(key);
    return fs.existsSync(path) ? Util.toGetValue(fs.readJsonSync(path)) : defaultValue;
  }
```

**Comment by philcockfield** [comment]
Hmm, yes I think I agree.  The better option would be to return a `defaultValue` passed in a second parameter.

**Comment by philcockfield** [comment]
Published to NPM version [2.4.4](https://www.npmjs.com/package/file-system-cache/v/2.4.4)
